altsfw-USETHISONE/altsfw/QuizViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  QuizViewController.swift
+//  altsfw
+//
+//  Created by Allison Fain on 10/14/22.
+//
+
+import UIKit
+import SwiftUI
+
+var swipeTracker = SwipeTracker()
+class QuizViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+        let swipeRight = UISwipeGestureRecognizer(target: self, action: #selector(swipeFunc))
```

Comment:
great work, this looks good to me. can add an enum for dummy styles (or client specific if received)